app/client.py

```diff
@@ -0,0 +1,74 @@
+import requests
+from pprint import pprint
+
+
+class Client:
+    def __init__(self, host):
+        self.host = host
+        self.token = None
+
+    def signup(self, username, password, account_name):
+        data = {
+            "data": {
+                "attributes": {
+                    "username": username,
+                    "password": password,
+                    "account_name": account_name
+                }
+            }
+        }
+        response = requests.post(f"{self.host}/auth/signup", json=data)
+        if response.status_code == 201:
+            self.token = response.json()["data"]["attributes"]["token"]
+            return self.token
+        else:
+            return response.json()
+
+    def login(self, username, password):
+        data = {
+            "data": {
+                "attributes": {
+                    "username": username,
+                    "password": password
+                }
+            }
+        }
+        response = requests.post(f"{self.host}/auth/login", json=data)
+        if response.status_code == 201:
+            self.token = response.json()["data"]["attributes"]["token"]
+            return self.token
+        else:
+            return response.json()
```

Comment:
It is not convenient to handle errors from the client. Create some custom TaskExecutorClientError that is raised when API returns an error. Also, you can parse an error and make this exception very handy:
```
try:
    client. signup(...)
except TaskExecutorClientError as err:
    if len(err.errors) == 1 and err.errors[0].loc == "account_name":
         LOG.error("Someone uses my account name!!")
         try_to_create_account_with_another_name()
```